sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/mlflow-model-local-inference-test.ipynb

```diff
@@ -0,0 +1,246 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load the MLFlow model locally and try predictions\n",
```

Comment:
I think the conda.yaml doesn't work as is on Windows. We need to add a note here (in all the task folders) that they need to do some additional steps to get it working locally in Windows.

In point 2., before 2.1 point, we can add the following:
Note: On Windows, installation of Pycocotools and recordclass packages from conda is not supported. Please remove "Pycocotools" and "recordclass" lines from conda.yaml before running the below steps.

Reply:
@gauravrajguru Do you think that should be sufficient ?

Reply:
Please sync with Gaurav on this. We also need to add c++ build tools to the warning message.